metrics_utility/management/commands/build_report.py

```diff
@@ -171,6 +185,7 @@ def _handle(self, *args, **options):
 
     def handle(self, *args, **options):
         try:
+            handle_env_validation('build')
```

Comment:
aah, nope, this can't be called before `init_logging()` in `handle`, otherwise throwing an exception causes the handler to fail on `AttributeError: 'Command' object has no attribute 'logger'`

fixing in #146